### PR TITLE
Added library list that support ReactiveCocoa

### PR DESCRIPTION
**Pull request by Legoless**
A couple of libraries already support ReactiveCocoa and for others there are a couple of extensions available. This pull request adds a list of libraries that I could find open source online.

### CODE REVIEW

**Comment by jspahrsummers** [comment]
As discussed in #1113, we'd only be interested in listing or “endorsing” libraries for which we've vetted the code quality. That's not to say that anything in this list _wouldn't_ meet that standard, but without @ReactiveCocoa/reactivecocoa collaborators spending the time to investigate, it's hard to say.

@kastiglione You expressed interest in helping out with this sort of thing, but that was almost a year ago now. Would you still be up for it?

**Comment by Legoless** [comment]
Understood, I can have a look as well. For starters, we could filter out those that are missing the Pods and that were not updated in last year or so. For standards, is there any document I could refer to?

**Comment by jspahrsummers** [comment]
Sorry, no. It's not just code style that we're interested in, but higher-level architectural patterns and “soundness,” which is why I would really like ReactiveCocoa collaborators to be involved.

**Comment by Legoless** [comment]
Okay, cool. Let me know if there is anything I can do.

**Comment by kastiglione** [comment]
I'll try to take a look. What are the thoughts of coverage completeness, for those libraries that are strictly wrappers? Do we care if they don't cover the full API, or do we only care that the parts that are covered, are covered fairly idiomatically?

**Comment by jspahrsummers** [comment]
> Do we care if they don't cover the full API, or do we only care that the parts that are covered, are covered fairly idiomatically?

I don't mind not wrapping the full API. It should absolutely be idiomatic, though.

File: Documentation/Libraries.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+iOS community develops many fantastic libraries to help developers create beautiful, quality applications. Wouldn't it be even better if libraries supported ReactiveCocoa out of the box?
+Certain libraries do not support the framework directly, but wrappers and extensions do exist.
+
+*Below is the list of libraries that support ReactiveCocoa in alphabetical order:*
+
+- [AFNetworking](https://github.com/AFNetworking/AFNetworking)
+  - [AFNetworking-RACExtensions](https://github.com/CodaFi/AFNetworking-RACExtensions)
+  - [AFNetworking-RACRetryExtensions](https://github.com/Legoless/AFNetworking-RACRetryExtensions)
+- [DANReactiveLocationManager](https://github.com/wearebase/DANReactiveLocationManager)
```
Comment (kastiglione) [inline Documentation/Libraries.md]: :x:

File: Documentation/Libraries.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- [EPSReactiveList](https://github.com/ElectricPeelSoftware/EPSReactiveList)
+- [Expecta](https://github.com/specta/expecta)
+  - [Expecta-ReactiveCocoa](https://github.com/kylef/Expecta-ReactiveCocoa)
+- [JSONModel](https://github.com/icanzilb/JSONModel)
+  - [JSONModel-RACExtensions](https://github.com/Legoless/JSONModel-RACExtensions)
```
Comment (kastiglione) [inline Documentation/Libraries.md]: :heavy_multiplication_x: All operators are library specific versions of `-tryMap:`.

File: Documentation/Libraries.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+- [Kiwi](https://github.com/kiwi-bdd/Kiwi)
+  - [ReactiveKiwi](https://github.com/chrisdevereux/ReactiveKiwi)
+- [MEReactiveFoundation](https://github.com/willbur1984/MEReactiveFoundation)
```
Comment (kastiglione) [inline Documentation/Libraries.md]: :heavy_multiplication_x:  The name "Reactive" is a red herring. Has nothing to do with RAC.

File: Documentation/Libraries.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+- [MEReactiveKit](https://github.com/TeamMaestro/MEReactiveKit)
```
Comment (kastiglione) [inline Documentation/Libraries.md]: :heavy_multiplication_x:  Primary not about RAC, though it does use RAC. There are only two classes which expose RAC APIs, neither of which are considerable.

File: Documentation/Libraries.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+- [MMPReactiveCoreLocation](https://github.com/mpurbo/MMPReactiveCoreLocation)
```
Comment (kastiglione) [inline Documentation/Libraries.md]: :heavy_multiplication_x: Non-idiomatic.

File: Documentation/Libraries.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+- [ReactiveHealthKit](https://github.com/kerryknight/ReactiveHealthKit)
+- [ReactiveMediaPlayer](https://github.com/zakdances/ReactiveMediaPlayer)
+- [ReactiveNetService](https://github.com/chrisdevereux/ReactiveNetService)
+- [ReactiveNSXMLParser](https://github.com/aceontech/ReactiveNSXMLParser)
+- [ReactiveViewModel](https://github.com/ReactiveCocoa/ReactiveViewModel)
+- [RNCryptor](https://github.com/RNCryptor/RNCryptor)
+  - [ReactiveCryptor](https://github.com/ndouglas/ReactiveCryptor)
+- [SBLReactiveLocation](https://github.com/sbl/SBLReactiveLocation)
+- [STMReactiveReachability](https://github.com/stefanomondino/STMReactiveReachability)
+- [UIGestureRecognizer-RACExtension](https://github.com/kaiinui/UIGestureRecognizer-RACExtension)
```
Comment (kastiglione) [inline Documentation/Libraries.md]: :heavy_multiplication_x: Redundant to RAC's support of gesture recognizers.

File: Documentation/Libraries.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+- [ReactiveDribbbleKit](https://github.com/liyong03/ReactiveDribbbleKit)
+- [ReactiveHealthKit](https://github.com/kerryknight/ReactiveHealthKit)
+- [ReactiveMediaPlayer](https://github.com/zakdances/ReactiveMediaPlayer)
+- [ReactiveNetService](https://github.com/chrisdevereux/ReactiveNetService)
+- [ReactiveNSXMLParser](https://github.com/aceontech/ReactiveNSXMLParser)
+- [ReactiveViewModel](https://github.com/ReactiveCocoa/ReactiveViewModel)
+- [RNCryptor](https://github.com/RNCryptor/RNCryptor)
+  - [ReactiveCryptor](https://github.com/ndouglas/ReactiveCryptor)
+- [SBLReactiveLocation](https://github.com/sbl/SBLReactiveLocation)
+- [STMReactiveReachability](https://github.com/stefanomondino/STMReactiveReachability)
```
Comment (kastiglione) [inline Documentation/Libraries.md]: :heavy_multiplication_x: Non-idiomatic.

**Comment by Legoless** [comment]
So should we update this and remove libraries? But if you are going to keep very strict rules about this, is almost pointless to have any list at all.

**Comment by robb** [comment]
:-1: on the list, seems like the time could be better spent elsewhere and incentivize people to get their name on the list for fake internet points.

**Comment by ashfurrow** [comment]
All internet points are fake, @robb.

**Comment by Legoless** [comment]
I agree with @ashfurrow this is by no way meant to improve anyone's reputation. I did not even include any names. I just thought it was a nice addition to have this somewhere, it would definitely help me and probably some other developers as well. I understand all your points, so feel free to reject this, I just didn't think it would hurt anyone if this was in the main repo. :)

**Comment by ashfurrow** [comment]
I think having a list of example uses of ReactiveCocoa would be a great help to beginners. I know the (few) that were available when I was learning were very helpful.

**Comment by kastiglione** [comment]
> very strict rules about this, is almost pointless to have any list at all.

@Legoless Other than for "completeness" what value is there in listing libraries that we have to warn against?

**Comment by kastiglione** [comment]
I'm with @ashfurrow. If we can have resources to learn from, that's good. If "fake points" encourage "real" quality, that's fucking great trade off! :smile:

**Comment by ashfurrow** [comment]
@kastiglione I wish I could favourite GH comments.

**Comment by kastiglione** [comment]
Me too. Where can we find a GitHubber with pull around here. :star2:

**Comment by robrix** [comment]
:-1: to playing favourites with comments. All of your comments are equally beautiful :heart:

**Comment by Legoless** [comment]
@kastiglione Well, on the other side, if for nothing else, at least us beginners could learn how **not** to do things. ;)

**Comment by RuiAAPeres** [comment]
I think it would be useful and educational to layout some tips/patterns/recommendations when creating a wrapper.

**Pull request closed without merging**